### DISCUSSION TITLE
Build failing after deleting build folder

### DISCUSSION THREAD

**Discussion by ahives** [cake-build/cake#3386] [Q&A]
I am using CakeFrosting for the first time and when I executed my build.cake script on Windows I saw a build folder. When I open this folder there is a corresponding project. I did not think that I needed this because I had already written a build.cake script file so I deleted it. Now builds are failing during compilation with the below error:

MSBUILD : error MSB1009: Project file does not exist.
Switch: build/Build.csproj

I followed the below steps to bootstrap Cake from https://cakebuild.net/docs/getting-started/setting-up-a-new-project

I simply want to use Cake to build locally. Please Help.

**Comment by augustoproiete** [comment]
@ahives Do you have a solution file (`.sln`)? It's possible that the solution file is still referencing the `build/Build.csproj` project, that is now deleted. If you open the solution on Visual Studio and delete the `build/Build.csproj` from it and save, it should resolve the issue.

If it's not the above, it's possible that your `build.cake` file is trying to build this project in a different way. Please share the contents of your `build.cake` file if that's the case.

**Reply by ahives** [reply]
I found the issue. The build.ps1 script simply runs the Build.csproj, which I deleted. I was expecting that Cake Frosting would work like the "old" build.cake script. Now that I executed the template again and made my changes to Program.cs file everything seems to work fine. Thanks again.
